driver-core/src/test/java/com/datastax/driver/core/ProtocolVersionRenegotiationTest.java

```diff
@@ -16,42 +16,53 @@
 package com.datastax.driver.core;
 
 import com.datastax.driver.core.exceptions.UnsupportedProtocolVersionException;
+import com.datastax.driver.core.utils.CassandraVersion;
+import org.testng.SkipException;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import static com.datastax.driver.core.ProtocolVersion.*;
 import static org.assertj.core.api.Assertions.assertThat;
 
-@CreateCCM(CreateCCM.TestMode.PER_METHOD)
 public class ProtocolVersionRenegotiationTest extends CCMTestsSupport {
 
+    private ProtocolVersion protocolVersion;
+
+    @BeforeMethod(groups = "short")
+    public void setUp() {
+        protocolVersion = ccm().getProtocolVersion();
+    }
+
     /**
      * @jira_ticket JAVA-1367
      */
     @Test(groups = "short")
-    @CCMConfig(version = "2.1.16", createCluster = false)
     public void should_succeed_when_version_provided_and_matches() throws Exception {
-        Cluster cluster = connectWithVersion(V3);
-        assertThat(actualProtocolVersion(cluster)).isEqualTo(V3);
+        Cluster cluster = connectWithVersion(protocolVersion);
+        assertThat(actualProtocolVersion(cluster)).isEqualTo(protocolVersion);
     }
 
     /**
      * @jira_ticket JAVA-1367
      */
     @Test(groups = "short")
-    @CCMConfig(version = "3.6", createCluster = false)
-    public void should_fail_when_version_provided_and_too_low_3_6() throws Exception {
+    @CassandraVersion("3.0")
+    public void should_fail_when_version_provided_and_too_low_less_than_3_8() throws Exception {
+        if (ccm().getCassandraVersion().compareTo(VersionNumber.parse("3.8")) >= 0) {
+            throw new SkipException("CASSANDRA-11464 (3.8+) uses the attempted protocol version");
+        }
```

Comment:
Agreed that might be nice to have eventually as there will inevitably be some other places we'll need to do this.